### DISCUSSION TITLE
how to deploy Bolt 5 on a shared hosting ?

### DISCUSSION THREAD

**Discussion by Saint-Cyr** [bolt/core#3026] [Q&A]
While softaculous on A2 hosting is not supporting the vervsion 5, only manual installation is require but after doing:
1) Inside the `public_html` directory 
2) `composer create-project bolt/project myproject`
3) when loading the page, getting the message: Bolt - Installation error
If you can read this, you've (probably) installed Bolt in the wrong folder. It's recommended to install Bolt outside the so-called web root, ... 
3) When following any of the displayed instruction in the above error message, it does not work...

**Comment by I-Valchev** [comment]
Hi @Saint-Cyr,

I'd suggest running `composer create-project ...` outside of the public_html. Then, you can symlink public_html to bolt's `public` directory, and it should work :-)

**Reply by emilierabito** [reply]
Hello I'm stuck as well with OVH can someone help me ?? Thank you
